### PR TITLE
feat(core/deploy): UI widget for `Delay Before Scale Down`

### PR DESCRIPTION
**Pull request by ajordens**
image

### CODE REVIEW

**Comment by ajordens** [comment]
Basically a c&p of what normal red/black has and rolling red/black was missing.

I did add a bit of inline margining which could come out if we'd like consistent squishiness across the board.

image

File: app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/AdditionalFields.tsx [hunk 1]
```diff
@@ -44,6 +46,11 @@ export class AdditionalFields extends React.Component<IRollingRedBlackStrategyAd
     this.forceUpdate();
   };
 
+  private handleChange = (key: string, value: any) => {
```
Comment (erikmunson) [inline app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/AdditionalFields.tsx]: did typing `value` as a string blow up on you?

Comment (ajordens) [reply app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/AdditionalFields.tsx]: 🤷‍♀️ This was totally c&p from the redblack AdditionalFields.

I'll give it a whirl.

Comment (ajordens) [reply app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/AdditionalFields.tsx]: Gave it the `value: string` treatment

**Pull request closed without merging**